roles/rsyslog/tasks/inputs/basics/main.yml

```diff
@@ -7,22 +7,17 @@
   include_tasks:
     file: "{{ role_path }}/tasks/deploy.yml"
 
-- name: Create basics input configuration files in /etc/rsyslog.d
+- name: "Create basics input configuration file
+  input-basics-{{ _input.name }} in /etc/rsyslog.d"
   vars:
     __rsyslog_packages: []
     __rsyslog_rules:
-      - name: "input-basics-{{ item.name }}"
+      - name: "input-basics-{{ _input.name }}"
         type: input
-        state: "{{ item.state | d('present') }}"
+        state: "{{ _input.state | d('present') }}"
         sections:
           - options: "{{ lookup('template', __rsyslog_input_basics) }}"
   include_tasks:
     file: "{{ role_path }}/tasks/deploy.yml"
-  loop: '{{ logging_inputs | sort(attribute="type") }}'
```

Comment:
this is looping in sorted order, but the outer loop does not https://github.com/linux-system-roles/logging/blob/master/roles/rsyslog/tasks/main.yml#L296 - do you want the outer loop to be sorted?

Reply:
Thank you, @richm. I totally forgot about the special treatment for the basic input (imjournal), which conf file should be guaranteed there is just one file.